src/renderer/state.ts

```diff
@@ -219,6 +222,7 @@ export class AppState {
       isInstallingModules: observable,
       isKeepingUserDataDirs: observable,
       isOnline: observable,
+      isPublishingGistAsRevision: observable,
```

Comment:
IMO we should make this true by default since it's a low-impact change.

Reply:
I agree! The setting button is default checked to equal true